Add tab panels to TiDB Quick Start page and optimize content and structure (#5613) (#5619)
pingcap/docs: quick-start-with-tidb.md

1. Download and install TiUP:

    {{< copyable "shell-regular" >}}

    ```shell
    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
    ```

2. Declare the global environment variable:

    > **Note:**
    >
    > After the installation, TiUP displays the absolute path of the corresponding `profile` file. You need to modify the following `source` command according to the path.

    {{< copyable "shell-regular" >}}

    ```shell
    source .bash_profile
    ```

3. Start the cluster in the current session:

    - If you want to start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance, run the following command:

        {{< copyable "shell-regular" >}}

        ```shell
        tiup playground
        ```

    - If you want to specify the TiDB version and the number of the instances of each component, run a command like this:

        {{< copyable "shell-regular" >}}

        ```shell

>         tiup playground v5.0.1 --db 2 --pd 3 --kv 3 --monitor

Reviewer: ```suggestion
        tiup playground v4.0.12 --db 2 --pd 3 --kv 3 --monitor
```

        ```

>         The command downloads a version cluster to the local machine and starts it, such as v5.0.1. `--monitor` means that the monitoring component is also deployed.

Reviewer: ```suggestion
        The command downloads a version cluster to the local machine and starts it, such as v4.0.12. `--monitor` means that the monitoring component is also deployed.
```

        {{< copyable "shell-regular" >}}

        ```shell

> <<<<<<< HEAD
>         tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor
>         ```
> 
>         The command downloads a version cluster to the local machine and starts it，such as v4.0.11. `--monitor` means that the monitoring component is also deployed.
> =======
>         tiup playground v5.0.1 --db 2 --pd 3 --kv 3 --monitor
>         ```
> 
>         The command downloads a version cluster to the local machine and starts it, such as v5.0.1. `--monitor` means that the monitoring component is also deployed.
> >>>>>>> bcbe30736 (Add tab panels to TiDB Quick Start page and optimize content and structure (#5613))

Reviewer: ```suggestion
        tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor
        ```

        The command downloads a version cluster to the local machine and starts it，such as v4.0.11. `--monitor` means that the monitoring component is also deployed.
```
